assembly/index.ts

```diff
@@ -23,22 +23,22 @@ const kPtr = K.dataStart;
 
 //precomputed W for message block representing length 64 bytes for fixed input of 64 bytes for digest64
```

Comment:
Does this comment need to be updated?

Reply:
can change it to: precomputed W + K ....